easybuild/easyconfigs/l/libgcrypt/libgcrypt-1.10.1-GCCcore-11.3.0.eb

```
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'libgcrypt'
+version = '1.10.1'
+
+homepage = 'https://gnupg.org/related_software/libgcrypt/index.html'
+description = """Libgcrypt is a general purpose cryptographic library originally based on code from GnuPG"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://gnupg.org/ftp/gcrypt/%(name)s/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['ef14ae546b0084cd84259f61a55e07a38c3b53afc0f546bffcef2f01baffe9de']
+
+builddependencies = [('binutils', '2.38')]
+
+dependencies = [('libgpg-error', '1.46')]
+
+sanity_check_paths = {
+    'files': ['bin/libgcrypt-config', 'include/gcrypt.h', 'lib/libgcrypt.%s' % SHLIB_EXT],
+    'dirs': ['share']
+}
+
```

Since this installs a few binaries, it would be nice to add some simple sanity check commands for them, e.g.:
```suggestion

sanity_check_commands = [
    'dumpsexp --version',
    'hmac256 --version',
    'mpicalc --version',
    'libgcrypt-config --version | grep "%(version)s"',
]

```